bin/tessel-2.js

```diff
@@ -235,7 +236,10 @@ module.exports = function(args) {
 };
 
 module.exports.closeSuccessfulCommand = function() {
-  process.exit(0);
+  usb.terminateConnections()
```

Comment:
I'd prefer if we kept all tessel discovery and closing logic in `controller.js`. Could you move it to [`closeTesselConnections`](https://github.com/tessel/t2-cli/blob/master/lib/controller.js#L312) which gets called after every command?